File: Js/ejer22.js
let lado1 = 5;
let lado2 = 5;
let lado3 = 4;
if (lado1 == lado2 && lado2 == lado3){
  console.log ("El triangulo es equilatero");
}
 else if (lado1 == lado2 || lado1 == lado3 || lado2 == lado3 ) {
    console.log ("El triangulo es isosceles");
} 
else {
    console.log ("El triangulo escalano");
}

File: python/ejer22.py
lado1 = float(input("digite el primer lado: "))
lado2 = float(input("digite el segundo lado: "))
lado3 = float(input("digite el tercer lado: "))

if lado1 == lado2 and lado2 == lado3:
    print ("El triangulo es equilatero")
elif lado1 == lado2 or lado1 == lado3 or lado2 == lado3:
    print ("El triangulo es isosceles")
else: 
    print ("El triangulo escalano")    